en/sim_gazebo_gz/gazebo-models.md

```diff
@@ -0,0 +1,158 @@
+# Gazebo Models Repository (PX4-gazebo-models)
+
+The [PX4-gazebo-models](https://github.com/PX4/PX4-gazebo-models) repository is used to store all [Gazebo](../sim_gazebo_gz/README.md) models and worlds that are supported by PX4.
+
+- Models are stored in the `/models` directory and worlds are stored in the `/worlds` directory.
+- The [simulation-gazebo](https://github.com/PX4/PX4-gazebo-models/blob/main/simulation-gazebo) Python script is used for [starting Gazebo in standalone mode](../sim_gazebo_gz/README.md#standalone-mode).
+
+As PX4-gazebo-models is included in PX4 as a submodule, no additional work is required with a "normal" Gazebo `make` target, such as `make px4_sitl gz_x500`.
+
+For standalone builds you first have obtain the [simulation-gazebo](https://github.com/PX4/PX4-gazebo-models/blob/main/simulation-gazebo) script, and then it will fetch the models and worlds to `~/.simulation-gazebo` if that directory is not present.
+The build system won't automatically update the local copy again if the `.simulation-gazebo` folder is detected, but you can force it to update to the latest models and vehicles by passing the `overwrite` flag to the script. The resulting command will look something like: .
```

Comment:
Is the repetition with the sections below intended?

Reply:
@beniaminopozzan I am not quite sure what you are refering to here?

Reply:
I mean that
> For standalone builds you first have obtain the [simulation-gazebo](https://github.com/PX4/PX4-gazebo-models/blob/main/simulation-gazebo) script, and then it will fetch the models and worlds to `~/.simulation-gazebo` if that directory is not present.
The build system won't automatically update the local copy again if the `.simulation-gazebo` folder is detected, but you can force it to update to the latest models and vehicles by passing the `overwrite` flag to the script. The resulting command will look something like:

is quite similar to
>This will fetch the models and worlds from the [PX4 gazebo models repository](https://github.com/PX4/PX4-gazebo-models) into subfolders of the `~/.simulation-gazebo` directory the first time it is called (or more precisely, if the directory is not detected).
A _gz-server_ instance will then be launched using the default grey plane world.
The build system won't automatically update the local copy again if the `.simulation-gazebo` folder is detected, but you can force it to update to the latest models and vehicles by passing the `overwrite` flag to the script. The resulting command will look something like:

But nothing serius, just mentioning it.

Reply:
No this is valid. the second sentence is identical. I don't know how this happened but I will fix it.